packages/rocketchat-ui-message/message/popup/messagePopup.coffee

```diff
@@ -160,7 +160,7 @@ Template.messagePopup.onCreated ->
 
 	template.records = new ReactiveVar []
 	Tracker.autorun ->
-		if template.data.collection.find?
+		if template.data.collection.findOne?
 			template.data.collection.find().count()
```

Comment:
@marceloschmidt Yes, cuz array.find is valid too, see here https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find